.github/workflows/codecoverage.yml

```diff
@@ -34,48 +34,44 @@ jobs:
           coverage: xdebug
 
       - name: Checkout GitHub Pages branch for code coverage report
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           ref: gh-pages
           path: tests/reports/html
 
       - name: Install dependencies
-        run: composer install --prefer-dist --no-suggest --no-progress
+        run: composer update --prefer-dist --no-progress
```

Comment:
We have no lock file.

Reply:
Should we? I don't remember why not, except that to run the tests across the PHP matrix would need `composer update` if there was a lock file, I think.

Reply:
Composer is always reminding us

> No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.